elements/pf-dropdown/pf-dropdown-item.css

```diff
@@ -36,7 +36,8 @@
   display: none !important;
 }
 
-:host([disabled]) {
+:host([disabled]), 
+.disabled {
   pointer-events: none;
```

Comment:
[Screencast from 2024-03-28 09-18-25.webm](https://github.com/patternfly/patternfly-elements/assets/1466420/b75dd09b-b2a1-49a4-bc12-3480af14c6d6)
That pointer-events: none can be problematic if there are interactive controls deeper than the disabled item